PackageExplorer/Models/NuGetFrameworkModel.cs

```diff
@@ -0,0 +1,255 @@
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = ".NET for Windows Store apps", ShortName = "win8", Framework = Frameworks.FindByIdentifierShortName("win8").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Xamarin For iOS", ShortName = "xamarinios", Framework = Frameworks.FindByIdentifierShortName("xamarinios").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Xamarin For Android", ShortName = "MonoAndroid", Framework = Frameworks.FindByIdentifierShortName("MonoAndroid").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+		}
+	}
+
+	/// <summary>
+	/// Class NuGetTargetFrameworkItem.
+	/// </summary>
+	[PropertyChanged.ImplementPropertyChanged]
+	public class NuGetTargetFrameworkItem
+	{
+		/// <summary>
+		/// Gets or sets the name.
+		/// </summary>
+		/// <value>The name.</value>
+		public string Name { get; set; }
+
+		/// <summary>
+		/// Gets or sets the short name.
+		/// </summary>
+		/// <value>The short name.</value>
+		public string ShortName { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating whether this instance is default.
+		/// </summary>
+		/// <value><c>true</c> if this instance is default; otherwise, <c>false</c>.</value>
+		public bool IsDefault { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating whether this instance is selected.
+		/// </summary>
+		/// <value><c>true</c> if this instance is selected; otherwise, <c>false</c>.</value>
+		public bool IsSelected { get; set; }
+
+		/// <summary>
+		/// Gets or sets the selected item.
+		/// </summary>
+		/// <value>The selected item.</value>
+		public NuGetTargetFrameworkItem SelectedItem { get; set; }
+
+		/// <summary>
+		/// Gets or sets the framework.
+		/// </summary>
+		/// <value>The framework.</value>
+		public NuGetFramework Framework { get; set; }
+
+		/// <summary>
+		/// Gets or sets the items.
+		/// </summary>
+		/// <value>The items.</value>
+		public ObservableCollection<NuGetTargetFrameworkItem> Items { get; set; } = new ObservableCollection<NuGetTargetFrameworkItem>();
+
+		/// <summary>
+		/// Returns a <see cref="System.String" /> that represents this instance.
+		/// </summary>
+		/// <returns>A <see cref="System.String" /> that represents this instance.</returns>
+		public override string ToString()
+		{
+			return Name;
+		}
+	}
+
+	/// <summary>
+	/// Class NuGetTargetFrameworkItemExtensions.
+	/// </summary>
+	public static class NuGetTargetFrameworkItemExtensions
+	{
+		/// <summary>
+		/// Ases the targeted platform.
+		/// </summary>
+		/// <param name="items">The items.</param>
+		/// <returns>System.String.</returns>
+		public static string AsTargetedPlatformPath(this ObservableCollection<NuGetTargetFrameworkItem> items)
+		{
+			var selectedPlatforms = (from platform in items
```

Comment:
Not sure I understand -- there are two extension methods based on which type of collection is being used.  Where is the duplication?